packages/suite-desktop/src-electron/libs/app-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { app, RelaunchOptions } from 'electron';
+
+const isLinux = process.platform === 'linux';
+
+export const restartApp = () => {
+    const { logger } = global;
+
+    const options: RelaunchOptions = {};
+
+    options.args = process.argv.slice(1).concat(['--relaunch']);
+
+    if (isLinux && process.env.APPIMAGE) {
+        options.execPath = process.env.APPIMAGE;
+        options.args.unshift('--appimage-extract-and-run');
+    } else if (isLinux) {
+        options.execPath = process.execPath;
```

Comment:
are you sure that this is needed (or even reachable)  for the linux?

Reply:
Good point. Probably this else if is not required. I will check that. It is true that this "restart" fix was done because it was not working only on some specific Linux distributions. Probably only ones with set process.env.APPIMAGE.

Reply:
@matejkriz I will probably delete this whole restart fixing logic for Linux (Ubuntu, Mint)..., because after testing yesterday with @STew790, it looks like that on Electron v18 it stopped working and we need to find another fix. I will try to find out more on Monday

Reply:
removed and squashed